### ISSUE TITLE
create-near-app missing dev dependency asbuild

### ISSUE THREAD

**Issue by yerofey** [near/create-near-app#1260]
I tried to run `yarn dev` right after I installed all dependencies and it failed with this error:
<img width="929" alt="Снимок экрана 2021-09-08 в 01 14 53" src="https://user-images.githubusercontent.com/1276038/132417781-97fa7fea-c7e8-43ca-a6b4-98de1007d5f0.png">
But then I realised that I need 1 more dev dependency `asbuild`, after I added it all worked fine.
<img width="1437" alt="Снимок экрана 2021-09-08 в 01 15 19" src="https://user-images.githubusercontent.com/1276038/132417844-e1dc8187-4452-49d8-9e09-ed1aa04ca424.png">
<img width="1122" alt="Снимок экрана 2021-09-08 в 01 15 35" src="https://user-images.githubusercontent.com/1276038/132417866-19406ff7-4d32-47ad-9cdb-a87648902fc6.png">

Just in case - here are my NodeJS, NPM & Yarn versions:
<img width="301" alt="Снимок экрана 2021-09-08 в 01 17 15" src="https://user-images.githubusercontent.com/1276038/132417937-53d3177a-e295-46ec-8783-faf210e0b3da.png">

**Comment by gmihov001** [comment]
Just posting this for future readers' reference:
The above issue occurs when installing create-near-app with assembly script as an option. 
I got the same `asb: command not found` error. 
I read online that installing near-sdk-as would resolve that (the absence of `asbuild` commands), so I installed near-sdk-as, but somehow it gets installed in the `/contract` folder instead of the root. So I had to manually add the dependency info to the `package.json` in the root and run `yarn install` again. 
Only then the error was fixed.
